backend/clickhouse/traces.go

```diff
@@ -349,7 +352,7 @@ func (client *Client) TracesKeys(ctx context.Context, projectID int, startDate t
 		traceKeys = append(traceKeys, defaultTraceKeys...)
 	} else {
 		for _, key := range defaultTraceKeys {
-			if !strings.Contains(key.Name, *query) {
+			if strings.Contains(key.Name, *query) {
```

Comment:
Believe this was flipped?

Reply:
makes sense, also shouldnt this be `if strings.Contains(*query, key.Name) {`
to look for the default key in the query?

Reply:
Wait actually I think its currently right - for example, if the user types in "env" for `*query`, we would want to recommend the key, `environment`. So searching for the `*query` substring within the `key.Name`. Let me know if there is another case for this that I am missing